changelog/bug-1632708.md

```diff
@@ -0,0 +1,3 @@
+audience: general
+level: silent
+---
```

Comment:
It looks like the issue here was that it wanted a newline after the `---`
```
diff --git a/changelog/bug-1632708.md b/changelog/bug-1632708.md
index 1252e8ece..8006e8b3f 100644
--- a/changelog/bug-1632708.md
+++ b/changelog/bug-1632708.md
@@ -1,3 +1,3 @@
 audience: general
 level: silent
----
\ No newline at end of file
+---
```
fixed it.  I'll make a PR to be more lenient about that!